en/docs/install-and-setup/deploying-wso2-api-manager/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -300,4 +300,11 @@ Start WSO2 Identity Server for the changes to take effect. For more information,
 
         To overcome this issue, you need to create self-signed certificates for WSO2 API-M and WSO2 IS host names. Then [import the public certs]({{base_path}}/administer/product-security/General/UsingAsymmetricEncryption/admin-creating-new-keystores/#step-3-importing-certificates-to-the-truststore) of WSO2 API-M to the `trust-store.jks` of WSO2 IS and vice versa. This should resolve the SSL handshake failure.
 
+### Step 8 - Configure the other API-M components
 
```

Comment:
It is better if we can use the wording from [1] or similar, so that we can give the user the exact context.

[1] https://docs.wso2.com/display/AM260/Configuring+WSO2+Identity+Server+as+a+Key+Manager#ConfiguringWSO2IdentityServerasaKeyManager-Step8-ConfiguretheotherAPI-Mcomponents